### PR TITLE
c2c-32: Cleaned up emergency concepts

### PR DESCRIPTION
**Pull request by reagan-meant**
Changed the patient status to be coded.

### CODE REVIEW

File: configuration/concepts/emergency.csv [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+26ab29a5-234e-4798-96e1-a951d6b48f2b,,CIEL:159407,Poor,Poor,Poor,Poor,,,Misc,N/A,,,,,,,,,,,
+8a29e1fd-23b4-42ca-b420-10aa6c4a4e29,,CIEL:159405,Good,Good,Good,Good,,,Misc,N/A,,,,,,,,,,,
+d9a4fbbe-1dfa-4cad-98e2-0631b5d0fb1f,,CIEL:159406,Fair,Fair,Fair,Fair,,,Misc,N/A,,,,,,,,,,,
+8a67e468-0405-46e8-aab6-f64ce33fb85a,,CIEL:160429,Alive,Alive,Alive,Alive,,,Misc,N/A,,,,,,,,,,,
+38d18d84-6c4d-4fe8-a279-47a4e7629219,,CIEL:160432,Dead,Dead,Dead,Dead,,,Misc,N/A,,,,,,,,,,,
+5f55fe41-01aa-4a14-9c10-185f0ce71823,,CIEL:1067,Unknown,Unknown,Unknown,Unknown,,,Misc,N/A,,,,,,,,,,,
+a0daa3ea-a749-4f4a-afb9-e8ef922074b9,,CIEL:165870,Patient condition resolved,Patient condition resolved,Patient condition resolved,Patient condition resolved,,,Misc,N/A,,,,,,,,,,,
+5cc447e4-0ddc-43fd-b531-956629adede3,,CIEL:165871,Patient condition unresolved,Patient condition unresolved,Patient condition unresolved,Patient condition unresolved,,,Misc,N/A,,,,,,,,,,,
+014279e5-5a62-0b70-8105-a598bd6e483f,,CIEL:159640,Patient status,État du patient,Patient status,État du patient,,,Question,Coded,,Poor;Good;Fair;Alive;Dead;Unknown;Patient condition resolved;Patient condition unresolved,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/emergency.csv:13]: Let's rename to "Patient health status"

File: configuration/concepts/emergency.csv [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+7d15e31b-021f-9f99-50e6-50ace62f3962,,CIEL:164378,Action taken,Action posée,Action taken,Action posée,,,Misc,Text,,,,,,,,,,,
 6285ac8c-57e2-3190-7cf0-9e7773f522d0,,,Medication administered,Médication administrée,Medication administered,Médication administrée,,,Misc,Text,,,,,,,,,,,
-014279e5-5a62-0b70-8105-a598bd6e483f,,,Patient status,État du patient ,Patient status,État du patient ,,,Misc,Text,,,,,,,,,,,
-c47b5007-4ed6-1e55-215a-af7cbcb00329,,,Emergency Form,Formulaire d’ urgence,Emergency Form,Formulaire d’ urgence,,,ConvSet,N/A,,,Hour; Action taken; Medication administered; Patient status,,,,,,,,
+26ab29a5-234e-4798-96e1-a951d6b48f2b,,CIEL:159407,Poor,Poor,Poor,Poor,,,Misc,N/A,,,,,,,,,,,
+8a29e1fd-23b4-42ca-b420-10aa6c4a4e29,,CIEL:159405,Good,Good,Good,Good,,,Misc,N/A,,,,,,,,,,,
+d9a4fbbe-1dfa-4cad-98e2-0631b5d0fb1f,,CIEL:159406,Fair,Fair,Fair,Fair,,,Misc,N/A,,,,,,,,,,,
+8a67e468-0405-46e8-aab6-f64ce33fb85a,,CIEL:160429,Alive,Alive,Alive,Alive,,,Misc,N/A,,,,,,,,,,,
+38d18d84-6c4d-4fe8-a279-47a4e7629219,,CIEL:160432,Dead,Dead,Dead,Dead,,,Misc,N/A,,,,,,,,,,,
+5f55fe41-01aa-4a14-9c10-185f0ce71823,,CIEL:1067,Unknown,Unknown,Unknown,Unknown,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/emergency.csv:10]: Médiocre, Bien, Acceptable, Vivant, Mort, Inconnu.

File: configuration/concepts/emergency.csv [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+a0daa3ea-a749-4f4a-afb9-e8ef922074b9,,CIEL:165870,Patient condition resolved,Patient condition resolved,Patient condition resolved,Patient condition resolved,,,Misc,N/A,,,,,,,,,,,
+5cc447e4-0ddc-43fd-b531-956629adede3,,CIEL:165871,Patient condition unresolved,Patient condition unresolved,Patient condition unresolved,Patient condition unresolved,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/emergency.csv:12]: Let's remove those 2

File: configuration/concepts/emergency.csv [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
+014279e5-5a62-0b70-8105-a598bd6e483f,,CIEL:159640,Patient status,État du patient,Patient status,État du patient,,,Question,Coded,,Poor;Good;Fair;Alive;Dead;Unknown;Patient condition resolved;Patient condition unresolved,,,,,,,,,
+c47b5007-4ed6-1e55-215a-af7cbcb00329,,,Emergency Form,Formulaire d’ urgence,Emergency Form,Formulaire d’ urgence,,,ConvSet,N/A,,,Hour; Action taken; Medication administered; Patient status,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/emergency.csv:14]: Let's remove that set. Not needed in Forms 2.

File: configuration/concepts/emergency.csv [hunk 5]
```diff
@@ -1,6 +1,14 @@
 uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
 ca4a655a-8f16-53eb-0c0d-dbb6f58f3393,,,Hour,Heure,Hour,Heure,,,Misc,Text,,,,,,,,,,,
-7d15e31b-021f-9f99-50e6-50ace62f3962,,,Action taken,Action posée ,Action taken,Action posée ,,,Misc,Text,,,,,,,,,,,
+7d15e31b-021f-9f99-50e6-50ace62f3962,,CIEL:164378,Action taken,Action posée,Action taken,Action posée,,,Misc,Text,,,,,,,,,,,
 6285ac8c-57e2-3190-7cf0-9e7773f522d0,,,Medication administered,Médication administrée,Medication administered,Médication administrée,,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/emergency.csv:4]: This concept "Medication administered" should be a question, defined in the drugs csv.

**Review by rbuisson — CHANGES_REQUESTED** [review]
See comments

File: configuration/concepts/emergency.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/emergency.csv]: done

File: configuration/concepts/emergency.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/emergency.csv]: done

File: configuration/concepts/emergency.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/emergency.csv]: done

File: configuration/concepts/emergency.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/emergency.csv]: done

**Pull request merged**